cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -235,7 +235,7 @@ extractSyncOptions snp aop =
     iopts
       | enpOnlyGov snp = onlyGovInsertOptions useLedger
       | enpOnlyUTxO snp = onlyUTxOInsertOptions
-      | enpFullMode snp = fullInsertOptions useLedger
+      | enpFullMode snp = fullInsertOptions useLedger (enpKeepMetadataNames snp)
```

Comment:
This defaults to `[]`, so by default no metadata will be kept. This changes the dewfault behaviour, which should be to keep all.